styleguide/_themes/derek/scss/_modifiers.scss

```diff
@@ -16,20 +16,30 @@ $body-bg:  #fff !default;
 .bg-crumpled-paper { background-color: $light-gray; background-image: url(https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/elements/bg-crumpled-paper.png); }
 
 //centered-divs
-.center-block{ margin: 0 auto; float:none;}
+.center-block { margin: 0 auto; float: none; }
 
 //equal height columns
-.row-eq-height {
+.row-eq-height, .flex-container {
   display: -webkit-box;
   display: -webkit-flex;
   display: -ms-flexbox;
   display:         flex;
 }
 
+.flex-item {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display:         flex;
+  & > * {
```

Comment:
Is the objection with the vendor prefixes or the `& > *` wildcard selector?

In reading up on why the sass linter recommends against using vendor prefixes, it appears to recommend using a CSS post processor (Autoprefixer) or mixins from Compass or Boubon in addition to a pre processor (Sass). 

More info:
https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#vendorprefix

Correct me if I'm wrong @bighappyface, but I don't believe we have any of this available at this time and I'd imagine that would be a larger discussion to have down the road. Would it make sense to remove that rule from the linter for now and come back later?

As far as the wildcard selector goes, can you reference a specific child class instead (and also remove the nesting)? I don't really see the point of the wildcard here, but if you have a specific reason for it please let me know.